full-stack-tests/rpc/src/frontend/IpcWebSocket.test.ts

```diff
@@ -44,12 +43,5 @@ if (!ProcessDetector.isElectronAppFrontend) {
         resolve();
       });
     });
-
-    it("should not recurse in auth call", async () => {
-      await NativeApp.startup(socket);
-      IModelApp.authorizationClient!.onUserStateChanged.raiseEvent(new AccessToken(undefined, undefined, new Date(0)));
-      await NativeApp.callNativeHost("getAccessTokenProps");
-      IModelApp.authorizationClient = undefined;
-    });
```

Comment:
@calebmshafer Is there an equivalent for this test or no? Since we no longer have `onUserStateChanged`.